cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/image/AzureImageService.java

```diff
@@ -70,14 +72,18 @@ private AzureImage createImageAndNotify(AuthenticatedContext ac, VirtualMachineC
     }
 
     private Optional<VirtualMachineCustomImage> handleCustomImageCreationException(AzureImageInfo azureImageInfo, AuthenticatedContext ac,
-            AzureClient client, AzureManagedImageCreationCheckerContext checkerContext, CloudException e) {
+            AzureClient client, AzureManagedImageCreationCheckerContext checkerContext, CloudException originalException) {
         Optional<VirtualMachineCustomImage> customImage;
-        azureManagedImageCreationPoller.startPolling(ac, checkerContext);
+        Optional<Throwable> errorResult = azureManagedImageCreationPoller.startPolling(ac, checkerContext, false);
         customImage = findImage(azureImageInfo, client);
         if (customImage.isEmpty()) {
-            LOGGER.error("Failed to create custom image.", e);
             updateImageStatus(ac, azureImageInfo.getImageNameWithRegion(), azureImageInfo.getImageId(), CommonStatus.FAILED);
-            throw new CloudConnectorException(e);
+            if (errorResult.isPresent() && !(errorResult.get() instanceof TimeoutException)) {
```

Comment:
is there a particular reason why you decided not to catch TimeoutException in a try... catch here?